compiler/rustc_target/src/spec/abi.rs

```diff
@@ -8,24 +8,16 @@ mod tests;
 #[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Copy, Debug)]
 #[derive(HashStable_Generic, Encodable, Decodable)]
 pub enum Abi {
-    // N.B., this ordering MUST match the AbiDatas array below.
-    // (This is ensured by the test indices_are_correct().)
-
     // Multiplatform / generic ABIs
-    //
-    // These ABIs come first because every time we add a new ABI, we
-    // have to re-bless all the hashing tests. These are used in many
-    // places, so giving them stable values reduces test churn. The
-    // specific values are meaningless.
```

Comment:
Can this comment stay?

Reply:
Yes! Pardon, I'm still fixing things up at the moment. :sweat:

Reply:
This comment is now untouched in the relevant commit. (At the time of writing, this is 501c06e)